rootdir/vendor/etc/fstab.nile

```diff
@@ -2,9 +2,9 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/dev/block/bootdevice/by-name/system       /            ext4    ro,barrier=1                                                  wait,recoveryonly,verify,slotselect
```

Comment:
Is the removal of 'recoveryonly' intended (same as in following line)?

Reply:
Yes, it's something that blindly got copied and has no place in recent Android versions.